zis-aux/include/aux-manager.h

```diff
@@ -51,13 +51,18 @@ typedef struct ZISAUXCommArea_tag {
   int32_t flag;
 #define ZISAUX_HOST_FLAG_READY          0x00000001
 #define ZISAUX_HOST_FLAG_TERMINATED     0x00000002
+#define ZISAUX_HOST_FLAG_COMM_PC_ON     0x00000004
   uint32_t pcNumber;
   uint32_t sequenceNumber;
   int32_t termECB;
   uint64_t stoken;
   ASCB * __ptr32 ascb;
   uint16_t parentASID;
-  char reserved0[2];
+  uint8_t version;
```

Comment:
It's unfortunate the the version is positioned at the bottom of the structure. Can this be rolled into the EYECATHER?

Reply:
Yeah, we can take a risk and make this incompatible with the previous version... Let me think about this.

Reply:
Okay, I re-worked the communication area and changed the eyecatcher. This way the worst thing which may happen, if the old version is used, is the eyecatcher check will fail and the AUX A/S will terminate.